### ISSUE TITLE
Ghost videos in video list

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/studip-opencast-plugin#707]
Loading video list pages or clicking on upload causes the user interface to show a list of ghost videos which are then lost. Seeing that I was wondering why the UI would show me a number of events which will then be hidden from me. That is confusing.

https://github.com/elan-ev/studip-opencast-plugin/assets/1008395/c8c351dd-ed77-4926-bcd2-a80571fe1ac6

While this seems to be a half rendered list at first, this might also be dummy data or a dummy layout? In any case, this should not be there.

![Screenshot from 2023-06-01 23-55-49](https://github.com/elan-ev/studip-opencast-plugin/assets/1008395/deafcb7a-1a15-4ac3-a69f-c00afcdd7c63)

**Comment by tgloeggl** [comment]
Basically, this is a loading animation. The intentional use was not to have a blank page which then suddenly fills with videos in an effort to prevent empty states.
In case of the upload dialog, this is clearly inappropriate. So the enhancement should be to limit the usage of this.

There are a whole bunch of design changes suggested by a UI review i am implementing right of now. Lets take a look at this issue here afterwards again.

**Comment by alinastolzenburg** [comment]
Geringe Prio: Es wäre gut hier eine Animation einzusetzen die keine bestimmte Anzahl an Videos impliziert. Der nun links erscheinende "Ladekringel" --> zusätzlich zu dem Ghost empfinde ich diesen eher als verwirrender.

**Comment by tgloeggl** [comment]
The spinner is now gone an there is no more updating in the background, should therefore be much cleaner. If that is enough in your opinion, move it to "Done" otherwise reopen it. :)

**Comment by kai-wng-uos** [comment]
I still get the ghost videos. Since we have the progress bar, I think a blank page in the background would be less confusing.

https://github.com/elan-ev/studip-opencast-plugin/assets/152959695/bdd874d2-c4e4-454a-a18f-61ef971051f0

**Comment by tgloeggl** [comment]
Ok, then there should be no ghost videos when uploading a media, but otherwise they should still be there.

**Comment by kai-wng-uos** [comment]
I still get the ghost Video list right after the upload window closes. Since I also get the "in progress" video displayed in the list, which already lets me know that the processing has started, I feel like the ghost videos only make it more confusing.

**Comment by lkiesow** [comment]
I agree with Kai. The ghost videos popping up all the times lokk and feel very weird and I don't feel like anyone sees them as a loading screen.

If you want to indicate that something is being processed and people should wait, why not use something people are actually used to and will recognize as such.

For example a spinner like https://cssloaders.github.io/
